frontend/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,51 @@
+import { Link } from 'react-router-dom';
+import styles from './Footer.module.css';
+import logoPlaceHolder from '../../assets/logoPlaceHolder.png';
+
+export const Footer = () => {
+  return (
+    <div className="container">
+      <footer>
+        <div>
+          <img src={logoPlaceHolder} alt="logo" />
+        </div>
+        <div className={styles.footerInfoContainer}>
+          <div>
+            <Link
+              to="/terms-of-service"
+              className={styles.footerText}
+              id={styles.tos}
+            >
+              Terms of Service |
```

Comment:
you certainly can use the `|` char but I think most websites would render the `|` as a separate element altogether rather than this kind of trailing character behavior as text haha

Reply:
Made a span element with | as the text instead.